src/EnergyPlus/Material.cc

```diff
@@ -259,13 +260,14 @@ void GetMaterialData(EnergyPlusData &state, bool &ErrorsFound) // set to true if
             MaterNum = ip->getIDFObjNum(state, state.dataHeatBalMgr->CurrentModuleObject, counter);
 
             // Load the material derived type from the input data.
-            auto thisMaterial = state.dataMaterial->Material(MaterNum);
+            auto thisMaterial = new Material::MaterialChild;
```

Comment:
I know that plain `auto` works here, but please use `auto *` because otherwise it looks like something that should be an `auto &` if you know what I mean.

Reply:
yeah indeed. Forgot here. I'll change to auto *